PetrolEngine.cpp

```diff
@@ -1,7 +1,8 @@
 #include "PetrolEngine.hpp"
+#include "InvalidGear.hpp"
 #include <iostream>
 
-PetrolEngine::PetrolEngine(int power, float capacity, int gears)
+PetrolEngine::PetrolEngine(int power, float capacity, int gears=6)
```

Comment:
jesli chcecie miec domyslana wartosc dla parametru "gears" to musicie to zrobic w hederu

Reply:
Poprawione, wrzucilem defaulta do hedera.